hack/verify-test-featuregates.sh

```diff
@@ -41,19 +41,20 @@ if [[ -n "${direct_sets}" ]]; then
   rc=1
 fi
 
+function verify_no_new_unversioned_feature_gates {
+  local new_features_file=$1
+  local old_features_file="__masterbranch/${new_features_file}"
+  mkdir -p "$(dirname "${old_features_file}")"
+  git show master:"${new_features_file}" > "${old_features_file}"
+  go run test/static_analysis/main.go feature-gates verify-no-new-unversioned --new-features-file="${new_features_file}" --old-features-file="${old_features_file}"
+}
```

Comment:
nit: This doesn't seem to align with the name (and, presumably, the intended purpose) of this verify script since it was intended to verify _test_, where-as this verifies the use of non-test feature gates.

Reply:
The old https://github.com/kubernetes/kubernetes/blob/6a0aeb2adb8879dbdde7e06888aa91fed4526b8b/hack/verify-test-featuregates.sh#L46C45-L46C74 verifies non-test feature gates, so I just replace that one here. Maybe they should be moved to a new verify file?